frontend/src/main/components/BasicCourseSearch/CourseByInstructorSearchForm.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+import { quarterRange } from "main/utils/quarterUtilities";
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+
+const CourseByInstructorSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localStartQuarter = localStorage.getItem("CourseByInstructorSearch.StartQuarter");
+  const localEndQuarter = localStorage.getItem("CourseByInstructorSearch.EndQuarter");
+  const localInstructor = localStorage.getItem("CourseByInstructorSearch.Instructor");
+
+  const [startQuarter, setStartQuarter] = useState(localStartQuarter || quarters[0].yyyyq);
+  const [endQuarter, setEndQuarter] = useState(localEndQuarter || quarters[0].yyyyq);
+  const [instructor, setInstructor] = useState(localInstructor || "");
+    
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    fetchJSON(event, { startQuarter, endQuarter, instructor });
+  };
+
+  const handleInstructorOnChange = (event) => {
+    setInstructor(event.target.value);
+  };
+
+  // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
Two things:
* We can write tests for that; it's straightforward.  I'll show you how.
* even if we couldn't, disable all is way too much here.  You'll miss important other things.  You want to use the smallest exception possible if you are excluding code from mutation testing.

Reply:
Ah, I already wrote an article on this:

https://ucsb-cs156.github.io/topics/stryker/stryker_html_attribute_surviving_mutants.html